app/utils/di/SilhouetteModule.scala

```diff
@@ -35,13 +35,17 @@ class SilhouetteModule extends AbstractModule with ScalaModule {
     bind[UserService].to[UserServiceImpl]
     //bind[UserDAO].to[UserDAOImpl]
     bind[UserDAO].to[UserDAOSlick]
+    bind[AuthTokenService].to[AuthTokenServiceImpl]
+    //bind[AuthTokenDAO].to[AuthTokenDAOImpl]
+    bind[AuthTokenDAO].to[AuthTokenDAOSlick]
     bind[DelegableAuthInfoDAO[PasswordInfo]].to[PasswordInfoDAOSlick]
     bind[CacheLayer].to[PlayCacheLayer]
     bind[HTTPLayer].to[PlayHTTPLayer]
     bind[IDGenerator].toInstance(new SecureRandomIDGenerator())
     bind[PasswordHasher].toInstance(new BCryptPasswordHasher)
     bind[FingerprintGenerator].toInstance(new DefaultFingerprintGenerator(false))
     bind[EventBus].toInstance(EventBus())
+    bind[Clock].toInstance(Clock())
```

Comment:
Are we using this anywhere?

Reply:
No, I switched to using Java.sql.Timestamp and Instant